presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraSession.java

```diff
@@ -54,6 +54,8 @@ CassandraTable getTable(SchemaTableName schemaTableName)
 
     List<CassandraPartition> getPartitions(CassandraTable table, List<Object> filterPrefix);
 
+    boolean isMaterializedView(String caseInsensitiveSchemaName, String caseInsensitiveTableName);
```

Comment:
why not to use `SchemaTableName`? So far every name in Presto is case insensitive.